server/jetty/build.gradle

```diff
@@ -10,6 +10,8 @@ dependencies {
     Classpaths.inheritDagger(project, /* test */ true)
 
     Classpaths.inheritGrpcPlatform(project)
+    runtimeOnly project(':hotspot-impl')
+
```

Comment:
We can't have this unguarded - this was the part that was supposed to be in the hasProperty guard.